service/http/dffml_service_http/routes.py

```diff
@@ -131,10 +131,29 @@ async def get_mctx(self, request):
 
 
 class HTTPChannelConfig(NamedTuple):
+    """
+    Params:
```

Comment:
Thanks for documenting, but if we're documenting we need to stay in numpy format for consistency https://intel.github.io/dffml/contributing/style.html#docstrings